src/RecoveryServices/RecoveryServices.Backup.Providers/Providers/AzureFilesPsBackupProvider.cs

```diff
@@ -117,6 +117,11 @@ public RestAzureNS.AzureOperationResponse DisableProtectionWithDeleteData()
                                 resourceGroupName: vaultResourceGroupName);
         }
 
+        public RestAzureNS.AzureOperationResponse<ProtectedItemResource> UndeleteProtection()
```

Comment:
Can it be set as  a default implementation so that no duplicate codes in the derived classes?

Reply:
these are provider classes which implement interface methods. So it is not possible to implement a default interface method implementation